HISTORY.md

```diff
@@ -1,9 +1,11 @@
 # ripple-lib Release History
 
-## UNRELEASED
+## 0.17.8 (UNRELEASED)
```

Comment:
Should this be included?
https://github.com/ripple/ripple-lib/commit/2a90f9b134e168937dceb7da283d63734eac9e7c